src/models/OtrStore.js

```diff
@@ -0,0 +1,80 @@
+var _ = require("lodash");
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/**
+ * Stores OTR state for a given user
+ *
+ * Handles current sessions.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		sessions: {},
+		client: client,
+	});
+}
+
+/**
+ * Initializes an OTR session for query chans
+ *
+ * Event handler, to be called on chan creation
+ */
+OtrStore.prototype.initChan = function(args) {
+	var network_id = args.network;
+	var chan = args.chan;
+	if (chan.type === Chan.Type.QUERY) {
+		var network = _.find(this.client.networks, {id: network_id});
+		var session = this.getSession(chan.name, network);
+		var irc = network.irc;
+		if (session === undefined) {
+			session = new OTR({
+				/* After some tests, the only working value  with this lib and
+				 * node-otr4 lib as well
+				 */
+				fragment_size: 140,
+
+				/* after some tests, a good value not get kicked, and to be
+				 * fast enough, at least on FreeNode.
+				 * (tested pasting shout README, 742 chars).
+				 */
```

Comment:
I would actually comment on what the hell these values are/what these values are doing, not that "after some tests, these seem good" :D